project-koku/koku
# Adding status to ingress reports

## Jira Ticket

[COST-3544](https://issues.redhat.com/browse/COST-3544)

## Description

This change will add a status field to the posted ingress reports so we can update them on failed file processing.

## Testing

1. Checkout Branch
2. Restart Koku
3. Add a storage only source AWS/Azure/GCP
4. Post a report with Missing Required columns
5. We should gracefully bomb out + update the status field on the posted report

## Notes

...
